README.md

```diff
@@ -951,6 +973,9 @@ Other Style Guides
       const y = x + 1;
       return x * y;
     });
+    
+    // best
+    [1, 2, 3].map((x) => x * (x + 1));
     ```
```

Comment:
this is fine, but it may not actually be better - the "y" might add clarity, depending on what the code is doing.